packages/xrpl/test/models/utils.test.ts

```diff
@@ -308,9 +344,77 @@ describe('Models Utils', function () {
       const flagsMap = parseTransactionFlags(tx)
       assert.notStrictEqual(flagsMap, expected)
     })
+
+    it('parseTransactionFlags including GlobalFlag', function () {
+      const tx: PaymentChannelClaim = {
+        Account: 'r...',
+        TransactionType: 'PaymentChannelClaim',
+        Channel:
+          'C1AE6DDDEEC05CF2978C0BAD6FE302948E9533691DC749DCDD3B9E5992CA6198',
+        Flags: {
+          tfRenew: true,
+          tfClose: false,
+          // Global flag
+          tfInnerBatchTxn: true,
+        },
+      }
+
+      const expected = {
+        tfRenew: true,
+      }
+
+      const flagsMap = parseTransactionFlags(tx)
+      assert.notStrictEqual(flagsMap, expected)
```

Comment:
```suggestion
      const expected = {
        tfRenew: true,
        tfInnerBatchTxn: true
      }

      const flagsMap = parseTransactionFlags(tx)
      assert.strictEqual(flagsMap, expected)
```
`tfInnerBatchTxn: true` is missing in the `expected` value